version/version.go

```diff
@@ -34,20 +34,19 @@ func (i *ZreplVersionInformation) String() string {
 		i.Version, i.RuntimeGo, i.RuntimeGOOS, i.RuntimeGOARCH, i.RUNTIMECompiler)
 }
 
-var prometheusMetric = prometheus.NewUntypedFunc(
-	prometheus.UntypedOpts{
+var prometheusMetric = prometheus.NewGauge(
+	prometheus.GaugeOpts{
 		Namespace: "zrepl",
-		Subsystem: "version",
-		Name:      "daemon",
-		Help:      "zrepl daemon version",
+		Name:      "uptime",
+		Help:      "zrepl daemon start time and version",
```

Comment:
When I hear `uptime`, I expect the *duration* from the time since the thing in question started.
This metric here is not a duration but a timestamp, right?
In that case, `zrepl_start_time`?

Reply:
I did suggest `start_time` as a name. 😇
(one generally wants constant values as metrics, the uptime can easily be calculated in PromQL using `time()`)

Reply:
Sorry, should have listened more carefully :)